networkx/algorithms/__init__.py

```diff
@@ -51,7 +51,7 @@
 import networkx.algorithms.tree
 
 # bipartite
-from networkx.algorithms.bipartite import projected_graph, project, is_bipartite
+from networkx.algorithms.bipartite import projected_graph, project, is_bipartite, complete_bipartite_graph, random_graph
 # connectivity
```

Comment:
try to stick to the 80 columns per line. You can use parentesis for putting this in two lines.